routes/hotels.js

```diff
@@ -0,0 +1,62 @@
+const mongoose = require('mongoose')
+const express = require('express')
+const { Hotel } = require('../models/hotel')
+const router = express.Router()
+
+router.get('/', (req, res) => {
+  Hotel.find()
```

Comment:
Yyy, why do you have two routes from two different prs? It can cause problem after merging it to master. Now you can remove this folder or merge changes with await from your second pr